src/main/pages/che-7/end-user-guide/proc_mount-secret-as-a-environment-variable-in-workspace-container.adoc

```diff
@@ -0,0 +1,85 @@
+[id="mount-secret-as-a-environment-variable-in-workspace-container_{context}"]
+= Mount secret as a environment variable into workspace container
+
+
+This section describes how to mount an secret from user-s namespace as a environment variable(s) into a single or
+multiple workspace containers of {prod}.
+
+.Prerequisites
+
+* A running instance of {prod}. To install an instance of {prod}, see link:{site-baseurl}che-7/che-quick-starts/[{prod-short} 'quick-starts'].
+
+.Procedure
+
+* Create secret in user namespace following those rules :
+
+Labels must match the set of labels configured in `che.workspace.provision.secret.labels`
+property of {prod}. By default it is a set of two labels:
```

Comment:
I think it is good to explicitly tell what these labels are. And later provide some examples.